extensions/amp-social-share/1.0/amp-social-share.js

```diff
@@ -140,12 +198,14 @@ class AmpSocialShare extends PreactBaseElement {
           this.mutateProps(
             dict({
               'endpoint': expandedUrl,
+              'params': null,
               'target': target,
             })
           );
         } else {
           this.mutateProps(
             dict({
+              'endpoint': null,
```

Comment:
This is needed because sometimes we can switch between `custom-endpoint` or `pre-configured-endpoint` on attribute changes, so we need to also clear our the props used from the other type.